client-misk/src/test/kotlin/app/cash/backfila/client/misk/hibernate/SinglePartitionHibernateBackfillTest.kt

```diff
@@ -7,21 +7,22 @@ import app.cash.backfila.client.misk.embedded.BackfillRun
 import app.cash.backfila.client.misk.embedded.createDryRun
 import app.cash.backfila.client.misk.embedded.createWetRun
 import app.cash.backfila.client.misk.testing.assertThat
+import javax.inject.Inject
 import misk.hibernate.Id
 import misk.hibernate.Session
 import misk.hibernate.Transacter
 import okhttp3.internal.toImmutableList
 import okio.ByteString.Companion.encodeUtf8
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.Test
-import javax.inject.Inject
 
 abstract class SinglePartitionHibernateBackfillTest {
   @Inject @ClientMiskService lateinit var transacter: Transacter
   @Inject lateinit var backfila: Backfila
 
   @Test fun emptyTable() {
-    val run = backfila.createDryRun<SinglePartitionHibernateTestBackfill>()
+    // TODO: Does anyone know how I can reify without having to also add ShapeParameters?
+    val run = backfila.createDryRun<SinglePartitionHibernateTestBackfill, ShapeParameters>()
```

Comment:
I can't figure out how to reify the backfill and have that follow that its third parameter is something createDryRun should know about. Or is this impossible?

Reply:
pretty sure no

Reply:
I only need the type so I can have a parameter on function has to be the Parameters type.

Reply:
I decided it will just take Any for parameters. You could then technically use some other class if you wanted to.